### PR TITLE
Update eventing nightly and Install Eventing core TLS test resources

### PR DESCRIPTION
**Pull request by pierDipi**
Fixes https://github.com/knative-extensions/eventing-kafka-broker/pull/3869

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -515,3 +517,7 @@ function setup_kafka_channel_auth() {
       -p='[{"op": "remove", "path": "/data/auth.secret.ref.name"}, {"op": "remove", "path": "/data/auth.secret.ref.namespace"}]' || true
   fi
 }
+
+function install_eventing_core_test_tls_resources() {
+    ko apply -Rf "${repo_root_dir}/vendor/vendor/knative.dev/eventing/test/config/tls" || return $?
```
Comment (creydr) [inline test/e2e-common.sh:522]:
```suggestion
    ko apply -Rf "${repo_root_dir}/vendor/knative.dev/eventing/test/config/tls" || return $?
```

**Review by creydr — CHANGES_REQUESTED** [review]

File: test/e2e-common.sh (hunk 1 above)
Comment (pierDipi) [reply test/e2e-common.sh]: Thanks!

File: test/e2e-common.sh [hunk 2]
```diff
@@ -515,3 +517,7 @@ function setup_kafka_channel_auth() {
       -p='[{"op": "remove", "path": "/data/auth.secret.ref.name"}, {"op": "remove", "path": "/data/auth.secret.ref.namespace"}]' || true
   fi
 }
+
+function install_eventing_core_test_tls_resources() {
+    ko apply -Rf "${repo_root_dir}/vendor/knative.dev/eventing/test/config/tls" || return $?
```
Comment (creydr) [inline test/e2e-common.sh:522]:
```
Error from server (NotFound): error when creating "STDIN": namespaces "knative-eventing" not found
Error: error executing 'kubectl apply': exit status 1
```
Seems we need to create the namespace before

Comment (pierDipi) [reply test/e2e-common.sh:522]: Placed the call after installing Eventing core

**Review by Cali0707 — APPROVED** [review]
/lgtm

**Pull request merged**